review_practice/react-app/src/components/DatePicker/index.js

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from 'react';
+
+import 'react-dates/initialize';
+import 'react-dates/lib/css/_datepicker.css';
+import { DayPickerRangeController } from 'react-dates';
+
+import moment from 'moment';
+import 'moment/min/locales';
+
+import * as Style from './style';
+
+export default ({ requestToServer, datePicked, deleteButtonHandle, localeLanguage }) => {
+    moment.locale(localeLanguage);
+
+    const [startDate, setStartDate] = useState(null);
+    const [endDate, setEndDate] = useState(null);
+    const [focusedInput, setFocusedInput] = useState('startDate');
+
+    const onDatesChange = ({ startDate, endDate }) => {
+        datePicked(startDate, endDate);
+
+        setStartDate(startDate);
+        setEndDate(endDate);
+    };
+
+    const onFocusChange = focusedInput => {
+        if (focusedInput === null) setFocusedInput('startDate');
+        else setFocusedInput(focusedInput);
+    };
+
+    const saveButtonHandler = (startDate, endDate) => () => {
```

Comment:
```suggestion
    const saveButtonHandler = () => {
```

`DayPickerRangeController` 에서 `startDate` 와 `endDate` 를 받아오는것이 아니라서, 굳이 이렇게 감싸서 클로져 형태로 만들 필요는 없는 것 같아요.